mk/spksrc.tc-flags.mk

```diff
@@ -16,6 +16,14 @@ ifeq ($(strip $(TC_TYPE)),)
 TC_TYPE = DSM
 endif
 
+ifeq ($(strip $(TC_OS_MIN_VER)),)
+TC_OS_MIN_VER = $(firstword $(subst ., ,$(TC_VERS))).$(firstword $(subst ., ,$(subst $(firstword $(subst ., ,$(TC_VERS))),,$(TC_VERS))))-$(TC_BUILD)
+endif
```

Comment:
instead of firstword you may shorten this expression by using `word <n>`
you see such an expresstion in current cross/haproxy/Makefile.

Reply:
done